Dockerfile

```diff
@@ -1,26 +1,12 @@
-FROM golang:1.14 as build
-
-WORKDIR /iavl
+# This docker image is designed to be used in CI for benchmarks and also by developers wanting an environment that always has the lastest rocksdb and cleveldb. 
+FROM faddat/archlinux
```

Comment:
how about using one of the official arch or alpine linux distros?

Reply:
ping

Reply:
hey sure thing